README.md

```diff
@@ -1 +1,17 @@
-# wiki-framework
\ No newline at end of file
+# wiki-framework
+
+### Terminology Telegram Bot
+After activating virtual environment and cloning the repository:
+ 1. Go to */terminology_bot* directory.
+ 2. Execute *install.sh* script.
+
+    The script will:
+      - create Docker container with PostgreSQL,
+      - install the project dependencies from requirements.txt,
+      - execute *database.py* to create DB schema and insert initial data.
+
+ 3. Get your private token from Telegram BotFather (run */newbot* command and follow BotFather's instructions).
+ 4. Insert the *token* into the *config.ini* file.
+ 5. Set *multimedia_dir* in the *config.ini* file to store multimedia files for the terms.
```

Comment:
"." is added to path specified in config.ini. Shouldn't be added.

Reply:
Commit https://github.com/annkupriyanova/wiki-framework/pull/5/commits/f0df298c702feeea64ddd1cdf2bf3bd1733650f1: "." was deleted. Now you should set an absolute path to *multimedia_dir* in *config.ini*